src/pudl/transform/ferc1.py

```diff
@@ -1,11 +1,4 @@
-"""Routines for transforming FERC Form 1 data before loading into the PUDL DB.
-
-This module provides a variety of functions that are used in cleaning up the FERC Form 1
-data prior to loading into our database. This includes adopting standardized units and
-column names, standardizing the formatting of some string values, and correcting data
-entry errors which we can infer based on the existing data. It may also include removing
-bad data, or replacing it with the appropriate NA values.
-"""
+"""Classes & functions to process FERC Form 1 data before loading into the PUDL DB."""
```

Comment:
i would mention this module employs classes and design from `pudl.transform.classes`